recipes/openmp/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "openmp" %}
```

Comment:
Another consideration is the `intel` channel has `openmp`, which I think means Intel OpenMP. That said, I think there is a fair argument that they should reconsider that naming if they are going to be interacting with `conda-forge` and `defaults`.